app/imports/components/projects/ListProjects.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+
+export class ListProjects extends Component {
+
+  render() {
+    return(
+      <section id="projects-list" className="margin-bottom-basic">
+        <div className="grid-row">
+          <section className="grid-item item-s-12">
+            <h4 className="margin-bottom-tiny">Projects:</h4>
+          </section>
+        </div>
+        {this.props.projects.map((project, key) => (
+          <ListProject project={project} key={project._id} />
+        ))}
+      </section>
+    )
+  }
+
+}
+
+export class ListProject extends Component {
```

Comment:
Pretty common pattern to include child components in the same file. We need a convention to or not to.